webapp/src/serial.tsx

```diff
@@ -54,6 +55,7 @@ export class Editor extends srceditor.Editor {
 
     constructor(public parent: pxt.editor.IProjectView) {
         super(parent)
+        setInterval(this.dropStaleCharts.bind(this), 5000)
```

Comment:
should only be started be active while editor is active

Reply:
Fixed